### DISCUSSION TITLE
Record Type on Schema

### DISCUSSION THREAD

**Discussion by LucaGambaroPixart** [Automattic/mongoose#13677] [Q&A]
Hi, 
I need to implement somehow the `Record` type in some schemas.

I can't use `type: Map`, because I don't want to re-elaborate the object before returning the API response (the re-elaboration is mandatory otherwise the response will be an empty object, and also I don't want to change a lot of code).

Or, instead of records, there are some other ways to define a Schema with Objects without known keys? 
The must is to maintain the typing, so I don't want to set the type as `mongoose.Schema.Types.Mixed`.

Thanks

**Comment by vkarpov15** [comment]
`type: new Schema({}, { strict: false })` should work as well.
